treeopt/visualize.py

```diff
@@ -0,0 +1,370 @@
+                eqlines,
+                colors="black",
+            )
+        )
+
+        norm = colors.Normalize(vmin=v_min, vmax=v_max)
+        for im in images:
+            im.set_norm(norm)
+
+        images.append(
+            axes.scatter(
+                np.atleast_2d(self.opti_data.x[: self.opti_data.numDOE])[
+                    :, 0
+                ].reshape(-1),
+                np.atleast_2d(self.opti_data.x[: self.opti_data.numDOE])[
+                    :, 1
+                ].reshape(-1),
+                marker="x",
+                color="r",
+            )
+        )
+        images.append(
+            axes.scatter(
+                np.atleast_2d(self.opti_data.x[self.opti_data.numDOE :])[
+                    :, 0
+                ].reshape(-1),
+                np.atleast_2d(self.opti_data.x[self.opti_data.numDOE :])[
+                    :, 1
+                ].reshape(-1),
+                marker="x",
+                color="m",
+            )
+        )
+
+        fig.subplots_adjust(bottom=0.225, top=0.85)
+
+        ax2 = fig.add_axes([0.125, 0.075, 0.775, 0.075])
+
+        images.append(
+            mpl.colorbar.ColorbarBase(
+                ax2, cmap=mpl.cm.viridis, norm=norm, orientation="horizontal"
+            )
+        )
+
+        if hasattr(self.opti_data, "name"):
+            fig.suptitle(self.opti_data.name)
+
+        plt.show()
+
+    def plot_metamodell_nvars(self):
+        """
+        Generates a Corner Plot of the diffenent axes of the metamodell
+        :return: Nothing
+        :rtype: None
+        """
+
+        dim = self.opti_data.x.shape[1]
+
+        fig, axes = plt.subplots(nrows=dim, ncols=dim)
+
+        x_data = self.opti_data.x
+
+        x_step = np.zeros([self.num_ele, x_data.shape[1]])
+
+        for i in range(x_data.shape[1]):
+            x_step[:, i] = np.linspace(
+                self.opti_data.limits[i, 0],
+                self.opti_data.limits[i, 1],
+                self.num_ele,
+            )
+
+        XY = np.dstack(np.meshgrid(x_step))
+
+        fun1Vars = self.opti_data.sm.predict_values(XY.astype(np.float))
+
+        images = []
+        for i in range(x_data.shape[1]):
+            for j in range(i + 1, x_data.shape[1]):
+                images.append(
+                    axes[j, i].scatter(
+                        x_data[:, i], x_data[:, j], c="b", marker="o"
+                    )
+                )
+
+        if hasattr(self.opti_data, "name"):
+            fig.suptitle(self.opti_data.name)
+
+        plt.show()
+
+    def plot(self):
+        """
+        Calls a individual plot function depending on the provided data
+        :return: Nothing
+        :rtype: None
+        """
+
+        if (
+            self.opti_data.sim_keyword == "benchmarking"
+            or self.opti_data.sim_keyword == "Benchmarking"
+        ):
+            if self.opti_data.x.shape[1] == 1:
+                print("Hier ist eine 2-D Benchmarking Funktion zu sehen")
```

Comment:
1. German,
2. print statement

Reply:
deleted the line and changed the NotImplementedError